src/formats/jsonLd.test.ts

```diff
@@ -20,11 +20,31 @@
 //
 
 import { jest, describe, it, expect } from "@jest/globals";
+import { Response } from "cross-fetch";
 import { foaf, rdf } from "rdf-namespaces";
 import { DataFactory } from "n3";
-import jsonld from "jsonld";
+
+// FIXME: Move this to jest.config.ts when merging with the universal fetch PR
+import type * as RDF from "@rdfjs/types";
+import type { IQuadTerms } from "jest-rdf/lib/matchers/toBeRdfDatasetMatching";
+import matchers from "jest-rdf/lib/matchers";
```

Comment:
There isn't a submodule export map in the `jest-rdf` library `package.json`, so this is relying on internals and non-public API, so I would very much prefer not to introduce this. I see you've opened a related issue on the upstream repo, let's see if we can get that merged, and in the meantime let's just work around the matcher.